doc/guides/test/sanitycheck.rst

```diff
@@ -390,6 +395,17 @@ harness_config: <harness configuration options>
              tags: sensors
              depends_on: i2c
 
+    The following is an example yaml file with pytest harness_config options, default pytest_root
+    name will be used if pytest_root not specified.
+
+    ::
+
+        tests:
+          test:
+            harness: pytest
+            harness_config:
+              pytest_root: [pytest directory name]
```

Comment:
Since you added the option for specifying an individual timeout for a pytest test, it would be beneficial if you add its usage in this example.

Reply:
timeout removed in #33227